third_party/ssl/ssl/ssl_tls1_svr.c

```diff
@@ -150,7 +150,7 @@ static int ICACHE_FLASH_ATTR process_client_hello(SSL *ssl)
 
     offset += id_len;
     cs_len = (buf[offset]<<8) + buf[offset+1];
-    offset += 3;        /* add 1 due to all cipher suites being 8 bit */
```

Comment:
And this is wrong, as later in the code they check 16 bit ciphers. And since they missed high byte of the chiper, they check low byte of chiper + high byte of next cipher, which is wrong.